src/macaron/policy_engine/prelude/policy.dl

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved. */
+/* Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/. */
+
```

Comment:
Docs missing.

Reply:
Fixed in https://github.com/oracle-samples/macaron/commit/7584d191a925261dd109b81512af671aa81d42aa